EBC-Git-101/README.md

```diff
@@ -233,7 +233,8 @@ And of course, branch `issue1_text` has not been changed on the server repo. The
 
 ![33_master](https://cloud.githubusercontent.com/assets/5516900/16191065/b9cc9cfe-36e2-11e6-87ea-d8f3d675996a.gif)
 
-Now our local working copy is back on the master, enabling us to pull the server's master:
+Now our local working copy is back on the master, enabling us to pull the server's master.
+Do not get confused by git's answer: `your branch is up-to-date with 'origin/master'.`. Your local repository is not aware of the changes you just did on the server repo, so this is a comparison between your working copy (you just checked out) and the local master. Like for many other actions, it's highly recommended to start a `git pull` only with a clean working copy. This means that you should not have any uncommitted local changes before you pull.
```

Comment:
@mahoffmann 
Thanks for the suggestion, I think this is really an important point to add. I would agree to the first part in line 237 

>Do not get confused by git's answer: `your branch is up-to-date with 'origin/master'.`. Your local repository is not aware of the changes you just did on the server repo, so this is a comparison between your working copy (you just checked out) and the local master.

The second part is written in a broader sense and touches further topics:

>Like for many other actions, it's highly recommended to start a `git pull` only with a clean working copy. This means that you should not have any uncommitted local changes before you pull

I would suggest to stick to the original point and replace the second part with something like 

> To update your repo to the latest information on the server, use `git fetch origin`. If you follow this with a `git status` request, git should report with a new message informing you that your local copy is behind the server's version.